schemas/ispyb/stored_programs/sp_insert_dc_plan.sql

```diff
@@ -0,0 +1,100 @@
+DELIMITER ;;
+CREATE OR REPLACE DEFINER=`ispyb_root`@`%` PROCEDURE `insert_dc_plan`(
+	OUT p_id int(11) unsigned,
+	p_diffactionPlanId int(10),
+	p_name varchar(20),
+	p_experimentKind enum('Default','MXPressE','MXPressO','MXPressE_SAD','MXScore','MXPressM','MAD','SAD','Fixed','Ligand binding','Refinement','OSC','MAD - Inverse Beam','SAD - Inverse Beam','MESH','XFE','Stepped transmission','XChem High Symmetry','XChem Low Symmetry','Commissioning'),
```

Comment:
I suggest to remove the enum and replace it with a varchar(25) (if 25 is enough?). It's sufficient that the underlying table enforces the allowed values here, we don't need the enforcement in two places. In the future we might add more options to the enum, but we don't want to have to change the stored procedure as well.